linera-views/src/scylla_db.rs

```diff
@@ -646,17 +662,46 @@ impl ScyllaDbStoreInternal {
             .known_node(store_config.uri.as_str())
             .build()
             .await?;
-        let result = session.query("DESCRIBE KEYSPACE kv", &[]).await?;
-        let mut tables = Vec::new();
-        if let Some(rows) = result.rows {
-            for row in rows.into_typed::<(String, String, String, String)>() {
-                let value = row?;
-                if value.1 == "table" {
-                    tables.push(value.2);
+        let mesg_miss = "'kv' not found in keyspaces";
```

Comment:
`mesg_miss` -> Please let's use full English words in the right order. (Also, the usual abbreviation of message would be "msg".)

Reply:
Ok, changed to msg and order of words changed.